reference/OpenAPI.json

```diff
@@ -40075,6 +40151,47 @@
           "default_code_config"
         ]
       },
+      "ManagementProjectsSideKeys": {
+        "type": "object",
+        "title": "Project and Side Keys",
+        "description": "Returns `app_id` and `app_token` for the server-side and client-side authentication.",
+        "properties": {
+          "server_side_key": {
+            "type": "object",
+            "title": "`server_side_key`",
```

Comment:
Skoro tytułów nie powinniśmy potem zmieniać, to proponuję kosmetyczną zamianę (podobnie dalej):
`Server-side key`

Reply:
sprawdze czy wplywa to na sdk i podmienię